docs/storage.rst

```diff
@@ -108,7 +108,13 @@ On CentOS and Red Hat systems, and other RPM based distributions the command is
 
 .. code-block:: console
 
+  sudo yum install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-centos96-9.6-3.noarch.rpm
   sudo yum install postgresql-server postgresql
```

Comment:
Do we require this packages also as they are not mentioned in section **Installing PostgreSQL 9.6** of docs/building_fledge/building_fledge.rst?